lib/checkout_sdk/payments/payment_recipient.rb

```diff
@@ -6,7 +6,10 @@ module Payments
     #   @return [String]
     # @!attribute account_number
     #   @return [String]
+    # @!attribute address
+    #   @return [CheckoutSdk::Common::Address]
     # @!attribute zip
+    #   @deprecated Please use {#CheckoutSdk::Common::Address.zip} instead
```

Comment:
I would recommend to not add this annotation, because this class is shared with previous version, and that field is not deprecated on previous
![image](https://user-images.githubusercontent.com/30295119/215960544-6a093d1a-d8fe-4679-9aab-c44142250aad.png)